fvm/evm/handler/addressAllocator.go

```diff
@@ -11,10 +11,11 @@ import (
 
 const (
 	ledgerAddressAllocatorKey = "AddressAllocator"
-	uint64ByteSize            = 8
-	// addressIndexShuffleSeed is used for shuffling address index
-	// shuffling index is used to make address postfixes look random
-	addressIndexShuffleSeed = uint64(0xFFEEDDCCBBAA9987)
+	// `addressIndexMultiplierConstant` is used for mapping address indices
+	// into deterministic random-looking address postfixes. 
+	// The constant must be an ODD number. It is a "nothing-up-my-sleeves" constant.
+	// Look at `mapAddressIndex` for more details.
+	addressIndexMultiplierConstant = uint64(0xFFEEDDCCBBAA9977)
```

Comment:
What's the reasoning for the change from 0xFFEEDDCCBBAA9987 to 0xFFEEDDCCBBAA9977?

Reply:
I subjectively thought it looks more like a "nothing up my sleeve" constant this way (we couldn't use `88` because it's even to we jumped to `77`).  `87` is fine too.

Reply:
Ah, I see now 👍

Reply:
https://github.com/onflow/flow-go/pull/5344/commits/4fa6fdaee518c124646a098a2aef5e922413dc40